src/Fixer/Alias/ModernizeStrposFixer.php

```diff
@@ -320,7 +320,7 @@ private function isOfHigherPrecedence(Token $token): bool
             T_SR,                  // >>
         ];
 
-        static $operatorsPerContent = [
+        return $token->isGivenKind($operatorsKinds) || $token->equalsAny([
```

Comment:
Removed the `$operatorsPerContent` static variable but kept the `$operatorsKinds` above.
The content in `$operatorsKinds` is non-static because of the global constants without leading `\` or import.
Comparison for arrays containing global constants: https://3v4l.org/obMcg

Reply:
@gharlan , can you check if #8837 would bring any performance impact?

Reply:
thanks @gharlan 

applied, can you sync your PR with master?